src/layer/vector/Renderer.js

```diff
@@ -34,7 +34,11 @@ export var Renderer = Layer.extend({
 		// @option padding: Number = 0.1
 		// How much to extend the clip area around the map view (relative to its size)
 		// e.g. 0.1 would be 10% of map view in each direction
-		padding: 0.1
+		padding: 0.1,
+
+		// @option tolerance: Number = 0
+		// How much to extend click tolerance round a path/object on the map
+		tolerance : 0
```

Comment:
Shouldn't the default of this be `Browser.touch ? 10 : 0`?

Reply:
I think that we should try to build so little logic as possible round browser.touch in the library, because it's difficult to detect if it's a touch device or not. But still give the options for the users to be able to set tolerance. What do you think?

Reply:
Since I've supervised this PR, I think I will have to agree with @Muscot :smile: 

@IvanSanchez from a backwards compatibility point of view, you _are_ right, but I think the old default does not make sense: SVG does not have the same (larger tolerance when `touch` is true), and actually setting the tolerance to anything other than `0` will cause problems with lines/polygons that are right next to each other (see #5144).

So, I'd argue actually defaulting to `0` is what we've always should have done.

Reply:
If @perliedman says it's right to break a bit of backwards compatibility, then I have no problem with that.